### DISCUSSION TITLE
Panda Theme Switcher example

### DISCUSSION THREAD

**Discussion by HasanHaghniya** [chakra-ui/panda#1331] [Q&A]
hey guys, I am using panda in storybook and nextjs

i could not find any full example that how should i write my theme switch with panda , docs are not clear for me

i defined my semantic tokens and i guess i have some problem with data theme attribute in html tag

anyone already done this? any full example or repo? 

thanks

**Comment by anubra266** [comment]
@SecretKeeper The basic technique is to use classes. Have a `.light` or `.dark` classname on your html tag, depending on the color mode.
If you use nextjs, this package might be helpful
https://www.npmjs.com/package/next-themes
